src/main/java/com/example/demo/model/Book.java

```diff
@@ -0,0 +1,23 @@
+package com.example.demo.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Don't use @Data on JPA entities, they contain @ToString and @EqualAndHashcode that can be problematic, use @ Getter @Setter here